Common/src/main/resources/assets/tipsmod/lang/ja_jp.json

```diff
@@ -0,0 +1,81 @@
+{
+    "tipsmod.title.default": "ヒント",
+    "tipsmod.tip.no_tips": "ヒントが読み込まれていません。 構成を見直してください。",
+    "tipsmod.tip.undead_damage": "スケルトンのようなアンデッド モンスターは、ヘルス ポーションでダメージを受け、ダメージ ポーションで回復します。",
+    "tipsmod.tip.poison_immune": "ゾンビ、スケルトン、スパイダーはすべて毒ダメージを受けません。",
+    "tipsmod.tip.witch_conversion": "村人は雷に打たれると魔女になります。",
+    "tipsmod.tip.four_eggs": "ニワトリの卵を孵化させると、まれに 4 羽のニワトリがスポーンすることがあります。",
+    "tipsmod.tip.nether_lava": "溶岩はネザーでより速く広がります。",
+    "tipsmod.tip.east": "太陽、月、星はすべて東に向かって移動します。",
```

Comment:
Could you update this tip to reflect the changes from https://github.com/Darkhax-Minecraft/Tips/commit/ce8238f6a04c96f325357af1dfe5c52eed8a73c7